verl/trainer/config/actor/megatron_actor.yaml

```diff
@@ -78,7 +78,19 @@ megatron:
 
   # additional transformer config like: num_layers_in_first(/last)_pipeline_stage
   # oc.select: default val for ref.megatron.override_transformer_config
-  override_transformer_config: {}
+  override_transformer_config:
+    # Recompute configuration, same as in megatron.training.arguments
```

Comment:
seems it is duplicated with `gradient_checkpointing_kwargs ` https://github.com/volcengine/verl/blob/73d8a9dc60c7db50af7c14928e1bc26a7a5242b1/verl/trainer/config/ppo_megatron_trainer.yaml#L44-L56. 

To have the same interface with fsdp, do you think we should keep the `enable_gradient_checkpointing` or move the `override_transformer_config`?